README.md

```diff
@@ -176,4 +176,4 @@ To check that formatting is valid without actually making changes, run `npm run
 2. Determine the current version of the published stable documentation, e.g. at https://github.com/Qiskit/qiskit/releases
 3. Run `npm run gen-api -- -p <pkg-name> -v <version>`, e.g. `npm run gen-api -- -p qiskit -v 0.45.0`
 
-For `qiskit`, you can also regenerate historical versions by adding `--historical` to the arguments. This is useful when we improve our generation script.
+For `qiskit`, you can also regenerate historical versions by adding `--historical` to the arguments. This is useful when we improve our generation script.
```

Comment:
Nit, it's better to avoid changing files with unrelated changes. I suspect the issue is if you're deleting the blank line on line 180. If so, adding that back will fix this.

PR tip: I like to review my own PRs and double check that every change I'm making is intentional.